popup.js

```diff
@@ -130,10 +130,19 @@ const mouseEvents = (
 const clearStorageURL = (tabUrl) => {
   chrome.storage.local.get("mappings", ({ mappings }) => {
     if (tabUrl && mappings[tabUrl]) {
-      mappings[tabUrl] = {
-        remove: [],
-        refine: [],
-      };
+      delete mappings[tabUrl];
+      chrome.storage.local.set({ mappings: mappings });
+      window.location.reload();
+    }
+  });
+};
+
+const clearAllStorageURL = () => {
+  chrome.storage.local.get("mappings", ({ mappings }) => {
+    if (mappings) {
+      Object.keys(mappings).forEach((tabUrl) => {
+        delete mappings[tabUrl];
+      });
       chrome.storage.local.set({ mappings: mappings });
```

Comment:
Can we replace these four lines by the following:
```suggestion
      chrome.storage.local.set({ mappings: {} });
```
This will avoid the loop. Let me know if this method will cause any issue.

Reply:
Good idea. This will be more efficient.